packages/frontend-2/pages/server-management/pending-invitations.vue

```diff
@@ -0,0 +1,272 @@
+<template>
+  <div>
+    <Portal to="navigation">
+      <HeaderNavLink
+        :to="'/server-management'"
```

Comment:
`to="/server-management"/` (check these everywhere)

Reply:
Can you confirm this one for me, removing the single quotes breaks this for me.